test/cpp/end2end/BUILD

```diff
@@ -195,6 +195,33 @@ grpc_cc_binary(
     ],
 )
 
+grpc_cc_test(
+    name = "client_fork_test",
+    size = "enormous",
+    srcs = ["client_fork_test.cc"],
+    external_deps = [
+        "absl/strings",
+        "gtest",
+    ],
+    # TODO(hork): locate and fix the 1.5% flake in the poll implementation
```

Comment:
Does this hang? If so, we should likely set a shorter timeout since this tests' size is `enormous`.

Reply:
It does not, apparently in 5000 runs. It should have that bug in WorkStealingThreadPool where quiesce and fork are called concurrently from different threads. But it looks like it's not running into it. So I removed the `flaky = True` and `size = "enormous"` for now.